WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -82,6 +81,16 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
         return currentPreferredStatusBarStyle
     }
 
+    override var hidesBottomBarWhenPushed: Bool {
+        set {
+
+        }
+
+        get {
+            true
+        }
+    }
+
```

Comment:
Maybe condense this down a few lines? 

```suggestion
    override var hidesBottomBarWhenPushed: Bool {
        set { }
        get { true }
    }
```